helm/charts/determined/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: determined
 description: A Helm chart for Determined
-version: "0.4.0"
+version: "0.5.0"
```

Comment:
blocking: I think this is going to conflict with #2101. @shiyuann can you advise on what to do here?

Reply:
@eecsliu yeah. you need to search for `/helm/determined-0.4.0.tgz` and bump that version in the doc.

Reply:
will do, thanks!